xchange-core/src/main/java/org/knowm/xchange/currency/Currency.java

```diff
@@ -307,6 +306,14 @@
   public static final Currency STORJ = createCurrency("STORJ", "Storj", null);
   public static final Currency MOD = createCurrency("MOD", "Modum", null);
 
+  // Ethereum ERC20 Tokens
+  public static final Currency DAI = createCurrency("DAI", "Dai", null);
+  public static final Currency WETH = createCurrency("WETH", "Wrapped Ether", null);
+  public static final Currency USDC = createCurrency("USDC", "USD Coin", null, "UDC");
+  public static final Currency PBTC = createCurrency("PBTC", "dydx BTC Perpetual (Linear)", null);
+  public static final Currency PLINK = createCurrency("PLINK", "dydx Link Perpetual (Linear)", null);
+  public static final Currency PUSD = createCurrency("PUSD", "dydx USD Perpetual (Inverse)", null);
```

Comment:
I am of the opinion that stablecoins should be in a stable coin section. Also I have mixed feelings of having perpetual contracts that belong to a single exchange in the currencies class.

@timmolter @walec51 @mdvx would love to get some input here.